scripts/copy_from_upstream/copy_from_upstream.yml

```diff
@@ -15,18 +15,18 @@ upstreams:
     git_branch: master
     git_commit: faf5c3fe33e0b61c7c8a7888dd862bf5def17ad2
     kem_meta_path: '{pretty_name_full}_META.yml'
-    common_meta_path: 'Common_META.yml'
+    #common_meta_path: 'Common_META.yml'
```

Comment:
Is this not needed any more? If so, why not delete?

Reply:
updated in 0b91245833c1bb9756519316fb13b8cb4d67d2cd